pages/home.html

```diff
@@ -0,0 +1,478 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet" />
+    <link rel="stylesheet" href="/src/input.css" />
+  </head>
+  <body class="relative w-full h-full font-roboto">
+    <nav class="flex flex-row justify-between items-center px-20 mt-3">
+      <h1 class="font-bold text-base">{finsweet</h1>
+      <div class="flex flex-row gap-8">
+        <p> <a href="./home.html"> Home</p>
+        <p><a href="./product.html">Product</a></p>
+        <p> <a href="./pricing.html"> Pricing</a></p>
+        <p> <a href="./aboutUs.html"> About us </a></p>
+        <p> <a href="./blog.html"> Blog </a></p>
+        <p> <a href="./contact.html"> Contact</a></p>
+        <button
+          class="rounded-lg text-white bg-black text-sm align-middle px-5 py-4 justify-center"
+        >
+          Free trial
+        </button>
+      </div>
+    </nav>
+    <div
+      class="absolute w-[524px] h-[496px] right-[566px] top-[313px] body-background -z-20"
+    ></div>
+    <div class="px-20 flex flex-col justify-center items-center mt-20">
+      <h1
+        class="font-roboto font-bold text-6xl leading-tight text-center text-slate-800"
+      >
+        The Best Software to Grow<br/> your Sales and Services
+      </h1>
+      <h1
+        class="mt-8 font-roboto font-normal text-lg leading-normal text-center"
+      >
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor<br/> invidunt ut labore et dolore magna aliquyam erat.
+      </h1>
+    </div>
+    <div class="flex justify-between items-center h-[72px] mx-auto bg-white w-[576px] mt-10 rounded-[20px] p-3">
+      <input
+        type="text"
+        placeholder="Enter your email"
+        class="bg-transparent font-roboto"
+      />
+      <button
```

Comment:
You are reapeating yourself for buttons, Try to create tailwind class
 which is reusable with modifiers